keras/layers/recurrent.py

```diff
@@ -316,7 +317,7 @@ class SimpleRNN(Recurrent):
     # References
         - [A Theoretically Grounded Application of Dropout in Recurrent Neural Networks](http://arxiv.org/abs/1512.05287)
     """
-
+    @interfaces.legacy_SimpleRNN_support
```

Comment:
Please use names that are stylistically correct function names, e.g. `legacy_simplernn_support`.